email-log.php

```diff
@@ -36,21 +36,74 @@
 	define( 'EMAIL_LOG_PLUGIN_FILE', __FILE__ );
 }
 
-// handle installation and table creation
+/**
+ * Handles installation and table creation.
+ */
 require_once plugin_dir_path( __FILE__ ) . 'include/install.php';
 
 /**
- * The main Plugin class
+ * The main plugin class.
+ *
+ * @since Genesis
  */
 class EmailLog {
+
+	/**
+	 * Stores the filesystem directory path with trailing slash.
+	 *
+	 * @since Genesis
+	 * @access public
+	 * @var string $include_path
+	 */
 	public $include_path;
 
+	/**
+	 * Stores the admin screen object.
+	 *
+	 * @since Genesis
+	 * @access private
+	 * @var string $include_path
+	 */
 	private $admin_screen;
 
+	/**
+	 * Stores version number.
```

Comment:
Generally just write about the value that it stored in the variable as comment instead of the fact that a value is stroed in the variable.

Eg: `Stores version number` should be just `Version number` or `Plugin version number`.
`Stores the admin screen object` can just be `Admin screen object`.